### ISSUE TITLE
"ValueError: Value is too big!" when importing stream in Blender

### ISSUE THREAD

**Issue by vinnividivicci** [specklesystems/speckle-blender#74]
### Expected vs. Actual Behavior

Expected the stream receive to work, but instead it crashes out saying this:

```
Error: Python: Traceback (most recent call last):
  File "C:\Users\vince\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\bpy_speckle\operators\streams.py", line 273, in execute
    stream_data = operations.receive(commit.referencedObject, transport)
  File "C:\Users\vince\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\modules\specklepy\api\operations.py", line 70, in receive
    return serializer.read_json(obj_string=obj_string)
  File "C:\Users\vince\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\modules\specklepy\serialization\base_object_serializer.py", line 242, in read_json
    return self.recompose_base(obj=obj)
  File "C:\Users\vince\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\modules\specklepy\serialization\base_object_serializer.py", line 300, in recompose_base
    base.__setattr__(prop, self.handle_value(value))
  File "C:\Users\vince\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\modules\specklepy\serialization\base_object_serializer.py", line 321, in handle_value
    obj_list = [self.handle_value(o) for o in obj]
  File "C:\Users\vince\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\modules\specklepy\serialization\base_object_serializer.py", line 321, in <listcomp>
    obj_list = [self.handle_value(o) for o in obj]
  File "C:\Users\vince\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\modules\specklepy\serialization\base_object_serializer.py", line 335, in handle_value
    return self.recompose_base(obj=obj)
  File "C:\Users\vince\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\modules\specklepy\serialization\base_object_serializer.py", line 260, in recompose_base
    obj = self.get_child(obj=obj)
  File "C:\Users\vince\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\modules\specklepy\serialization\base_object_serializer.py", line 353, in get_child
    return ujson.loads(ref_obj_str)
ValueError: Value is too big!


location: <unknown location>:-1
```

Number of elements in the stream: 2701
Original source: Revit 2022

Trying to receive stream in Blender 3.0 (MS App Store Version)
### Reproduction Steps & System Config (win)

It seems to work fine with other smaller models, so I'm not sure if this because of a specific element type or we are simply hitting a limitation somewhere in the sync process.

May be linked to the maximum size of JSON object because of this line:
`return ujson.loads(ref_obj_str)`

**Comment by izzylys** [comment]
hi @vinnividivicci,

funnily enough, this has nothing to do with the size of your data. this is due to you having an integer that is too long
https://github.com/ultrajson/ultrajson/issues/49

I'm not sure what this int could be, but it would be helpful if you'd be able to share this stream with me? I've not had this pop up before so I'd be curious to have a look

**Comment by vinnividivicci** [comment]
I sent you a DM through the community.

**Comment by izzylys** [comment]
this has been closed by specklesystems/specklepy#154 and should no longer be an issue! let me know if you are still running into anything similar 😊
